package.json

```diff
@@ -39,21 +39,33 @@
     "url": "https://github.com/uniiverse/milkyway/issues"
   },
   "homepage": "https://github.com/uniiverse/milkyway#readme",
-  "dependencies": {},
+  "dependencies": {
+  },
   "peerDependencies": {
+    "react": "^16.8.6",
```

Comment:
@snada does this make sense?

Reply:
Yes, this should be fine.

In terms of dependencies, semantic-ui-react [has a peerDependency for react](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/package.json#L186) at `^16.8.0`, while [our host repo has](https://github.com/uniiverse/host/blob/master/package.json#L39) `^16.9.0`.

So I _guess_ we will not get any warning, and even if that happens I'm pretty sure everything will work just fine.